src/Common/test/Common.TestResources/IgnoreLineEndingsComparer.cs

```diff
@@ -35,6 +35,8 @@ public bool Equals(string x, string y)
 
     public int GetHashCode(string obj)
     {
+#pragma warning disable S3900 // Arguments of public methods should be validated against null
         return obj.GetHashCode();
+#pragma warning restore S3900 // Arguments of public methods should be validated against null
```

Comment:
```suggestion
        return HashCode.Combine(obj);
```

This _is_ a valid warning. Instead of suppressing it, it can easily be addressed.